python/numpy-in-torch-datasets.yml

```diff
@@ -0,0 +1,22 @@
+rules:
+- id: numpy-in-torch-datasets
+  patterns:
+    - pattern-either:
+      - pattern: |
+          class $X(Dataset):
+            def __getitem__(...):
```

Comment:
It seems we only match `def __getitem__` if its first member of `class $X`, shouldn't there be `...` after `class $X` line?

Pls add testcases for those

Reply:
Oddly enough, it seems to work without it, but I went ahead and made the change in case. I'll go ahead and add a test case for different permutations of methods.